packages/www/components/AssetDetails/AssetPlayerBox/ProcessingProgress.tsx

```diff
@@ -1,5 +1,5 @@
 import { useMemo } from "react";
-import { Asset } from "livepeer";
+import { Asset } from "@livepeer.studio/api";
```

Comment:
Curious about this change, was the other version breaking your local builds?

Reply:
Yeah it was breaking everything when I was trying to get the `asset.source` - that type didn't have it and it was used in several places